canvas/services/event.py

```diff
@@ -31,11 +33,12 @@ def _get_error_messages(submissions):
     return error_messages
 
 
-def _get_submission_status(submissions):
+def _get_submission_status(submissions, students):
```

Comment:
the variable name `students` count probably be named better too

`students` kinda implies a list of Student, when you actually mean something more like `num_students`

Even better if you do `def _get_submission_status(submissions, num_students: int):`